ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc

```diff
@@ -310,19 +313,29 @@ TorchCore::validateOutputTensor (const at::Tensor output, unsigned int idx)
   at::Tensor sliced_output = output.slice (0);
   c10::IntArrayRef sliced_output_sizes = sliced_output.sizes ();
 
+  /** if idx is in bounds */
+  if (outputTensorMeta.num_tensors <= idx) {
+    ml_loge ("Invalid output meta: trying to access index %d with total %d tensors",
+        idx, outputTensorMeta.num_tensors);
+    ml_loge ("Update the number of outputs to >=%d in the model description", idx + 1);
```

Comment:
Please call ml_loge a single time for a single error. It's a simple stderr output now, but we can add more semantics with a ml_loge call later. (please simply merge the two strings into one.)

Reply:
done.